pkg/gardenlet/controller/seed/seed/reconciler_reconcile.go

```diff
@@ -1303,3 +1313,57 @@ func CleanupLegacyLokiPriorityClass(ctx context.Context, seedClient client.Clien
 
 	return client.IgnoreNotFound(seedClient.Delete(ctx, priorityClass))
 }
+
+// CleanupOldFluentBit deletes all old fluent-bit resources which are not installed by the fuent-operator
+func CleanupOldFluentBit(ctx context.Context, seedClient client.Client) error {
+	// Resources which does not duplicate these from the operators
+	uniqueResource := []client.Object{
+		&corev1.ConfigMap{ObjectMeta: metav1.ObjectMeta{Name: "fluent-bit-config", Namespace: v1beta1constants.GardenNamespace}},
```

Comment:
fluent-bit ConfigMaps are actually with unique names, aren't they? See
```
% k -n garden get cm | grep fluent-bit-config
fluent-bit-config-04aabdff                              10     5h45m
```

I think you should rather delete the ConfigMap via labelSelector.

Reply:
Hmmm then maybe we do not have to delete it as the GRM has the functionality to delete an unique ConfigMaps which are no longer in use?

Reply:
Yes, you are right. GRM's garbage collector should clean up this ConfigMap. 👍